platforms/android/default.config.py

```diff
@@ -0,0 +1,6 @@
+ABIs = [
+    ABI("2", "armeabi-v7a", None, 21, cmake_vars=dict(ANDROID_ABI='armeabi-v7a with NEON')),
+    ABI("3", "arm64-v8a",   None, 21, cmake_vars=dict('ANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON')),
```

Comment:
Makes sense for "Android NDK r27 and higher" according to documentation.

Why do not enable that by default in the main build_sdk.py script?

Reply:
I want to switch CI to the default configuration and always build SDK and AAR packages with 16kb pages support. The packages will work with both old and new memory configuration. It just tunes ELF sections alignment a bit.  The option will be ignored with NDK before 27.

Reply:
```suggestion
    ABI("3", "arm64-v8a",   None, 21, cmake_vars=dict(ANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES='ON')),
```